HMCL/src/main/resources/assets/lang/I18N_zh.properties

```
@@ -1266,6 +1266,7 @@ version.manage.redownload_assets_index=更新遊戲資源檔案
 version.manage.remove=刪除該實例
 version.manage.remove.confirm.trash=真的要刪除實例「%s」嗎? 你可以在系統的資源回收筒 (或垃圾桶) 中還原目錄「%s」來找回該實例。
 version.manage.remove.confirm.independent=由於該實例啟用了「(全域/實例特定) 遊戲設定 → 執行路徑 → 各實例獨立」設定，刪除該實例將導致該遊戲的存檔等資料一同被刪除！真的要刪除實例「%s」嗎?
+version.manage.remove.failed=刪除實例失敗，可能檔案被佔用。
```

查了好几篇文章，都存在混用「占/佔」的情况。  
也有一些文章认为两个词都能用。

不过目前 HMCL 繁体其余地方都使用「占」，且繁化姬的「台灣化」也转换为「占」。  
所以我希望这里先改成「占」。

```suggestion
version.manage.remove.failed=刪除實例失敗。可能檔案被占用。
```